deepreg/dataset/loader/interface.py

```diff
@@ -89,15 +89,26 @@ def get_dataset_and_preprocess(
         repeat: bool,
         shuffle_buffer_num_batch: int,
         data_augmentation: Optional[Union[List, Dict]] = None,
+        num_parallel_calls: int = tf.data.experimental.AUTOTUNE,
```

Comment:
As @fepegar has mentioned, are we sure we want this to be the default behaviour?

Reply:
ok then i change to 1

Reply:
actually, I believe `num_workers` limits the CPU globally, and this will not leads to use more CPUs, I just tested.